README.md

```diff
@@ -1,62 +1,26 @@
 # Conversations Demo Web Application Overview
-
 SDK version of this demo app: ![](https://img.shields.io/badge/SDK%20version-2.0.0-blue.svg)
 
 The latest available SDK version of this demo app: ![](https://img.shields.io/badge/SDK%20version-2.0.0-green.svg)
 
 ## Getting Started
-
 Welcome to the Conversations Demo Web application. This application demonstrates a basic Conversations client application with the ability to create and join conversations, add other participants into the conversations and exchange messages.
 
-What you'll minimally need to get started:
-
-- A clone of this repository
-- [A way to create a Conversations Service Instance and generate client tokens](https://www.twilio.com/docs/conversations/identity)
-
-## Building
-
-### Set the value of `REACT_APP_ACCESS_TOKEN_SERVICE_URL`
-
-Set the value of `REACT_APP_ACCESS_TOKEN_SERVICE_URL` in the `.env` file to point to a valid Access-Token server. If you don't yet have a token server you can follow the instructions [here](#for-testing-purposes-it-is-possible-to-create-a-simple-token-generator-using-a-twilio-function) to create one using a Twilio function. The instructions here are only meant for **demo/testing purposes** it is not a production ready server.
-
-Your Access-Token server should provide a valid token for valid credentials by URL:
-
- ```
-$REACT_APP_ACCESS_TOKEN_SERVICE_URL?identity=<USER_PROVIDED_USERNAME>&password=<USER_PROVIDED_PASSWORD>
- ```
-
-And return HTTP 401 in case of invalid credentials.
-
-If an `.env` file doesn't exist, create one with the following contents:
-```
-REACT_APP_ACCESS_TOKEN_SERVICE_URL=http://example.com/get-token/
-```
-
-NOTE: No need for quotes around the URL, they will be added automatically.
+What you'll need to get started:
```

Comment:
add an empty line before the bullet list

(some md renderers fail to render it as a list otherwise)